metrics/wait.io.table/table.go

```diff
@@ -264,9 +264,13 @@ func (t *Table) Collect(ctx context.Context, levelName string) ([]blip.MetricVal
 		tblName = *values[1].(*string)
 
 		for i := 2; i < len(cols); i++ {
+			counterType := blip.CUMULATIVE_COUNTER
+			if o.truncate {
```

Comment:
Can this logic be done once in `Prepare`?

Reply:
Sure, can do!